frontend/src/features/admin-form/common/components/AdminFormNavbar/AdminFormNavbar.tsx

```diff
@@ -60,6 +67,21 @@ export const AdminFormNavbar = ({
 }: AdminFormNavbarProps): JSX.Element => {
   const { ref, onMouseDown } = useDraggable<HTMLDivElement>()
   const { isOpen, onClose, onOpen } = useDisclosure()
+  const { pathname } = useLocation()
+
+  const tabResponsiveVariant = useBreakpointValue({
+    base: 'line-dark',
+    xs: 'line-dark',
+    lg: 'line-light',
+  })
+
+  const checkTabActive = useCallback(
+    (to: string) => {
+      const match = pathname.match(ACTIVE_ADMINFORM_BUILDER_ROUTE_REGEX)
+      return (match?.[2] ?? '/') === `/${to}`
```

Comment:
Clearly this should never happen, but maybe we should throw an error instead if we fail to match, to rule out unintended behavior?

```suggestion
      if (!match) throw new Error('Unexpected pathname')
      return (match[2] ?? '/') === `/${to}`
```

Reply:
this might happen on a form link without a slash at the end, that is why a default is needed